### ISSUE TITLE
fix keyboard navigation of topics autocomplete

### ISSUE THREAD

**Issue by hroberts** [mitmedialab/MediaCloud-Web-Tools#1439]
It is confusing that keyboard navigation does not work with the topics autocomplete box.

Even though I know it does not work, I still occasionally do something like:

* type 'immigration' into the search box
* wait for a list of results to appear
* hit the down arrow to get a visible highlight of the first result
* press 'enter'
* wait
* press 'enter' again
* remember that keyboard navigation doesn't work and click on the desired result

I just had to support a researcher who has assumed for months that the autocomplete doesn't work at all because he didn't get to the final step.

**Comment by rahulbot** [comment]
related to #1234
